app/components/Phone.js

```diff
@@ -0,0 +1,21 @@
+const React = require('react');
+const classnames = require('classnames');
+
+const Audience = React.createClass({
```

Comment:
...and actually, use the newer `defineSvg` construct that does away with some of the boilerplate.

e.g. https://github.com/Crowdtilt/tilt-images/blob/master/app/components/Envelope.js